src/gen/java/net/clesperanto/wrapper/kernelj.java

```diff
@@ -37,6 +37,7 @@ public static class Tier1 extends Pointer {
     }
 
     public static native void addImageAndScalar(@Const @ByRef ProcessorJ proc, @Const @ByRef BufferJ src, @Const @ByRef BufferJ dst, float scalar);
+    public static native void gaussianBlur(@Const @ByRef ProcessorJ proc, @Const @ByRef BufferJ src, @Const @ByRef BufferJ dst, float sigmaX, float sigmaY, float sigmaZ);
```

Comment:
This is auto-generated from JavaCPP, I am still questioning the meaning of it being in the repo but lets not care for now